extensions/core/interfaces/map/display.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+  <div class="readonly-map">
+    <div class="interface-map-wrap">
+      <div class="map-display" id="directusMapDisplay" :style="{width:options.width+'px',height:options.height+'px'}">
```

Comment:
`id="directusMapDisplay"`

Keep in mind that this interface can be rendered multiple times on the page, which will cause duplicate ID issues! Field names are unique and can be accessed through `this.name`

Reply:
So now we're not rendering the map on `display`, this is not an issue anymore. Just curious, `this.name` will return component name right? How will it be unique each time if same component is rendered multiple times? 🤔